extensions/wikia/ArticleComments/js/ArticleComments.js

```diff
@@ -123,7 +123,8 @@
 					id: commentId,
 					method: 'axEdit',
 					rs: 'ArticleCommentsAjax',
-					useskin: window.skin
+					useskin: window.skin,
+					cb: Date.now()
```

Comment:
@garthwebb: `AjaxResponse` is the legacy way of handling AJAX requests. It should emit response caching headers in a similar fashion as `WikiaResponse` - in this case `Cache-Control: private, s-maxage=0`. Please file a ticket for the Platform Team - we will fix it.